ocaml/xapi-blobstore/test/dune

```diff
@@ -0,0 +1,24 @@
+(executable
```

Comment:
```suggestion
(test
```

Reply:
fixed in a55dda750acb1428d04911cac3ac092a307f5ee9